docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -655,6 +655,64 @@ Sets the `Group` of `Issuer` to be used when issuing certificates with `cert-man
   </TabItem>
 </Tabs>
 
+## `tls.existingSecretName`
+
+| Type | Default value | Can be used in `custom` mode? | `teleport.yaml` equivalent |
+| - | - | - | - |
+| `string` | `""` | ✅ | `proxy_service.https_keypairs` |
+
+`tls.existingSecretName` tells Teleport to use an existing Kubernetes TLS secret to secure its web UI using HTTPS. This can be
+set to use a TLS certificate issued by a trusted internal CA rather than a public-facing CA like LetsEncrypt.
+
+You should create the secret in the same namespace as Teleport using `kubectl create secret tls my-tls-secret --cert=/path/to/cert/file --key=/path/to/key/file`
+
+See https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets for more information.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  tls:
+    existingSecretName: my-tls-secret
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```shell
+  --set tls.existingSecretName=my-tls-secret
+  ```
+  </TabItem>
+</Tabs>
+
+## `tls.existingSecretNameCA`
+
+| Type | Default value | Can be used in `custom` mode? |
+| - | - | - |
+| `string` | `""` | ✅ |
+
+`tls.existingSecretNameCA` sets the `SSL_CERT_FILE` environment variable to load a trusted CA or bundle in PEM format into Teleport pods.
```

Comment:
Is it just the Auth Service that reads this environment variable?

Also, will readers be expected to understand the use of `SSL_CERT_FILE` more than (or before they) understand `tls.existingSecretNameCA`? If not, I think we can make this paragraph clearer by removing the mention of the env var.

Reply:
> Is it just the Auth Service that reads this environment variable?

It's the proxy service that's potentially reading it here to establish a full trust chain when serving Teleport's TLS listener.

I mostly put this here for competent readers who might Ctrl+F SSL_CERT_FILE against the reference to see whether we have some way to override this. It's fairly well documented online that `SSL_CERT_FILE` and `SSL_CERT_DIR` are some ways to solve this - I figured it wouldn't hurt if people are deep enough into PKI.